core-blocks/latest-comments/index.php

```diff
@@ -5,9 +5,30 @@
  * @package gutenberg
  */
 
-// Used to get `_draft_or_post_title`, which prevents empty-title posts from
-// appearing with no title.
-require_once( ABSPATH . 'wp-admin/includes/template.php' );
+/**
+ * Get the post title.
+ *
+ * The post title is fetched and if it is blank then a default string is
+ * returned.
+ *
+ * Copied from `wp-admin/includes/template.php`, but we can't include that
+ * because:
+ *
+ * 1. it causes bugs
+ * 2. it's in the admin; ideally we *shouldn't*
+ *
+ * @since 3.3.0
+ *
+ * @param int|WP_Post $post Optional. Post ID or WP_Post object. Default is global $post.
+ * @return string The post title if set; "(no title)" if no title is set.
+ */
+function gutenberg_draft_or_post_title( $post = 0 ) {
+	$title = get_the_title( $post );
+	if ( empty( $title ) ) {
+		$title = __( '(no title)' );
```

Comment:
Text domain is missing

Reply:
Should be `__( '(no title)', 'gutenberg' );`, right? 😄